community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/fusion/FusionIndexAccessorTest.java

```diff
@@ -131,7 +131,9 @@ private void initiateMocks()
                 throw new RuntimeException();
             }
         }
-        fusionIndexAccessor = new FusionIndexAccessor( accessors, fusionVersion.selector(), indexId, mock( SchemaIndexDescriptor.class ), dropAction );
+        fusionIndexAccessor =
+                new FusionIndexAccessor( fusionVersion.slotSelector(), new Selector<>( accessors ), indexId, mock( SchemaIndexDescriptor.class ),
+                        dropAction );
```

Comment:
Can live on fewer lines

Reply:
IntelliJ multi-line formatting is very rarely optimal

Reply:
Problem is that it does not remove new lines, only add.